lessons/07-mcp/README.md

```diff
@@ -0,0 +1,482 @@
+# MCP, Model Context Protocol
+
+In this chapter you will learn:
+
+- How to use the Model Context Protocol, MCP to split up your server capabilities in a server and a client
```

Comment:
"in" --> "into"

How to use the Model Context Protocol, MCP to split up your server capabilities into a server and a client